cypress/e2e/feedPosts.spec.ts

```diff
@@ -1,3 +1,5 @@
+const postText = `Testing: ${Math.random().toString(36).substring(2, 8)}`;
+const commentText = `Testing: ${Math.random().toString(36).substring(2, 8)}`;
```

Comment:
generate a new comment inside of each `it` statement, that way if it[0] fails to delete then it[1] won't throw a false error